### ISSUE TITLE
Globbing imports inside partials

### ISSUE THREAD

**Issue by hinok** [jsahlen/gulp-css-globbing#15]
Hey,
Basically this plugin works great but problem occurs when some globbing has to be done in Sass partials which is latert imported in Sass file.

Files structure looks like:
```
├── app
│   ├── mixins
│   │   ├── _campaign-sales-box.scss
│   │   ├── _items-list.scss
│   │   ├── _pagination.scss
│   │   ├── _promoter.scss
│   │   ├── _promoters-count.scss
│   │   ├── _promoters-filters.scss
│   ├── _mixins.scss
│   ├── _base.scss
└── main.scss
```

*main.scss*
```
@import "app/base";
.foo { color: red; }
```

*app/_base.scss*
```
@import "mixins/*";
```

I've printed what returns `cssGlobbingPlugin`, finally it changes single @import to multiple @import but it looks like that change occures too late.

Compilation is finished with error
```
[gulp-sass] file to import not found or unreadable: mixins/*
```

**Comment by jsahlen** [comment]
Well, this isn't really a use case that `gulp-css-globbing` would be able to handle by itself. The plugin just changes the stream, and I'm guessing that `gulp-sass`/`libsass` does its own importing on the *files* specified. So to have globbing work in source files other than the entry-point, you would have to process the source files with `gulp-css-globbing`, write them to a temporary location, and then process the files from that temporary location with `gulp-sass`.

**Comment by hinok** [comment]
@jsahlen Good point. Now, It's not a big problem for me because I have small list of imports inside partials and I can handle them by myself, but it was quite strange for the first time that something works and few minutes later it doesn't work (when I've added imports inside partials).

I think that you could add information in README.md that imports inside partials are not allowed or something like this. Thanks for the response.

**Comment by ltarasiewicz** [comment]
I was also scratching my head not knowing why I cannot use globbing inside paritals that import other files.

**Comment by webegguk** [comment]
It'd be great to get a solution to this. Using a separate import to creat an ie8 specific style sheet with breakpoint and no query is what I would think is a common requirement and one I'm trying to solve whilst cutting compile time.

**Comment by jsahlen** [comment]
Hey guys. I'm pretty much not developing this lib more (as you may have noticed), but I'll happily accept pull requests for new features, or even transfer ownership if someone is feeling up to the task of maintaining it.
